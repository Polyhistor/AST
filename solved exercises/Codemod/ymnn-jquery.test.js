// codemod exercise 0 (ymnn-jquery)
// When you're finished with this exercise, run
//   "npm start exercise.codemod.1"
//   to move on to the next exercise

import pluginTester from 'babel-plugin-tester';
import ymnnJquery from './ymnn-jquery';

pluginTester({
  plugin: ymnnJquery,
  tests: [
    { code: 'foo.hide();', snapshot: false },
    { code: `$(el).hide();`, snapshot: true }
  ]
});
