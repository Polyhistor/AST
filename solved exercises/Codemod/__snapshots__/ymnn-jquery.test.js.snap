// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`2. ast-transform 1`] = `
"
$(el).hide();

      ↓ ↓ ↓ ↓ ↓ ↓

el.style.display = \\"none\\";
"
`;
